app/controllers/subscription_controller.rb

```diff
@@ -173,7 +173,7 @@ def multiple_add
   private
 
   def tag_selection_more_than_zero?
-    TagSelection.where(following: true, user_id: current_user.uid, tid: tag.tid).length.positive?
+    TagSelection.where(following: true, user_id: current_user.uid, tid: params[:tid]).length.positive?
```

Comment:
Is this broken before?

Reply:
No, it wasn't broken.
Instead it was showing the error `undefined local variable or method tag`. 
It was not able to recognize tag and hence the error. So, I got the value of `tid` from the parameter and it was working fine instead of defining it locally.

Reply:
Can you also paste the screenshot of the error you saw before and absence of it after the change? Thanks!

Reply:
@gauravano Attaching the screenshot
![Selection_074](https://user-images.githubusercontent.com/32747809/54554367-47ee2b80-49da-11e9-9231-ce87e8db1a3b.png)